docs/user_guide/mitigation/preprocessing.rst

```diff
@@ -20,23 +20,43 @@ Sensitive features can be correlated with non-sensitive features in the dataset.
 By applying the :code:`CorrelationRemover`, these correlations are projected away
 while details from the original data are retained as much as possible (as measured
 by the least-squares error). The user can control the level of projection via the
-:code:`alpha` parameter. In mathematical terms, assume we have the original dataset
-:math:`X` which contains a set of sensitive attributes :math:`S` and a set of
-non-sensitive attributes :math:`Z`. The removal of correlation is then
-described as:
+:code:`alpha` parameter.
+
+In mathematical terms, assume we have the original dataset :math:`X`, which
+contains a set of **sensitive attributes** denoted by :math:`S` and a set of
+**non-sensitive attributes** denoted by :math:`Z`. The goal is to remove
+correlations between the sensitive attributes and the non-sensitive attributes.
+
+Let :math:`m_s` and :math:`m_{ns}` denote the number of sensitive and non-sensitive
+features, respectively.
+Let :math:`\bar{S}` represent the mean of the sensitive attributes, *i.e.*,
+:math:`\bar{S} = (\bar{s}_1, \dots, \bar{s}_{m_s})`, where
+:math:`\bar{s}_j` is the mean of the :math:`j\text{-th}` sensitive feature.
+
+For each non-sensitive feature :math:`z_j` in :math:`Z`,
```

Comment:
Maybe we can say something like `\mathbf{z}_j\in\mathbb{R}^n` where `j=1,\dotsc,m_{ns}` to drive home the dimension of `\mathbf{z}_j`.